lib/generator.dart

```diff
@@ -53,23 +72,20 @@ NamedTypeNode _unwrapType(TypeNode type) {
 
 /// Generate a query definition from a GraphQL schema and a query, given
 /// Artemis options and schema mappings.
-QueryDefinition generateQuery(
+QueryDefinition _generateQuery(
```

Comment:
merging input files ends up requiring a lot of downstream logic changes, because of the 1-to-1 source file-to-operation assumption.